### DISCUSSION TITLE
Proof of your academic status delay (NOT PROCESSING)

### DISCUSSION THREAD

**Discussion by PhanNguyenDucTrung** [community/community#84817] [GitHub Education] [answered]
### Hi everyone,

I try to upload img to proof academic status but it seem like nothing happen, it look like the process stop or server is down
![Screenshot 2024-01-07 133313](https://github.com/community/community/assets/89243238/1970f124-c7a8-4291-b9ed-14e33c8d8f21)

**Comment by georgetoloraia** [comment]
TESTIG answer

**Reply by AnneRoki** [reply]
I am encountering the same problem and I have tried different image files and the problem still persists.

**Comment by everythinguwannatobe** [comment]
I understand that you're experiencing issues while trying to upload an image to prove your academic status. It seems like the process has stopped or the server may be down. To resolve this issue, I recommend reaching out to the support team of the website or platform where you're trying to upload the image. They should be able to assist you in troubleshooting the problem and ensuring that your academic status is properly verified.

**Comment by mishmanners** [comment, marked as answer]
The same is resolved here: https://github.com/orgs/community/discussions/88401#discussioncomment-8142514 and here: https://github.com/orgs/community/discussions/88446#discussioncomment-8148738

Please see these. The TL;DR:
- Refresh the page or clear your browser's cache and cookies
- Check for browser extensions that could be interfering with the JavaScript code and disable them
- Try a different browser
- Ensure your file less than 150kB
